cmd/csi-resizer/main.go

```diff
@@ -68,6 +68,8 @@ var (
 
 	handleVolumeInUseError = flag.Bool("handle-volume-inuse-error", true, "Flag to turn on/off capability to handle volume in use error in resizer controller. Defaults to true if not set.")
 
+	enableFSResizeAnnotation = flag.Bool("enable-fs-resize-annotation", true, "Flag to turn on/off capability to add an annotation to PV to denote its size prior to resize and to remove it upon completion of filesystem resize.")
+
```

Comment:
I think we wanted this as a feature gate and not a flag.  The k8s feature gates look like - `--feature-gate=RestoreDeletedVolumeSize`

Reply:
You can look at external-provisioner feature gate handling.

Reply:
Done :)